### DISCUSSION TITLE
Why auto-complete suggest me ARM32 if unavailable ?

### DISCUSSION THREAD

**Discussion by Et7f3** [community/community#26201] [Actions]
<p>Hello,</p>
<p>I’m little confused: we can see ARM32.</p>
<p><img src="https://cdn.discordapp.com/attachments/235200837608144898/680978782290444303/image0.jpg" alt="https://cdn.discordapp.com/attachments/235200837608144898/680978782290444303/image0.jpg"></p>
<p>and I get error when I use it. <img src="https://cdn.discordapp.com/attachments/235200837608144898/680978798425931788/unknown.png" alt="https://cdn.discordapp.com/attachments/235200837608144898/680978798425931788/unknown.png"></p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>The available GitHub-hosted runner types are:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/f7e95b2cc33259b6353a272a69041276e25386b6.png" data-download-href="/uploads/short-url/zn7THDRdNGuIfqFmtSnHDv9mFEy.png?dl=1" title="runs on.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181096044-7bb9da62-48f2-4d13-9014-ba3f0bc57826.png" alt="runs on.png" data-base62-sha1="zn7THDRdNGuIfqFmtSnHDv9mFEy" width="690" height="193" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/f7e95b2cc33259b6353a272a69041276e25386b6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">runs on.png</span><span class="informations">728×204 9.46 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Here is the document for runs-on: <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on" rel="nofollow noopener">https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on </a></p>
<p>ARM32 is for self-hosted runner. It could not be used alone. For example, it could be used like : </p>
<pre><code>runs-on: [self-hosted, linux, ARM32]
</code></pre>
<p>Please see more information in Using <a href="https://help.github.com/en/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow#using-self-hosted-runners-in-a-workflow" rel="nofollow noopener">self-hosted runners in a workflow</a>.</p>
